pkg/lib/serverTime.scss

```
@@ -1,3 +1,7 @@
 .ct-serverTime-time-picker {
   max-width: 7rem;
 }
+
+.ntp-server-input-group {
```

This should probably be:

```suggestion
.ntp-server-input-group:not(:last-child) {
```

...so it won't add the extra margin to the last item.